android/quest/src/main/java/org/smartregister/fhircore/quest/ui/shared/components/ServiceCard.kt

```diff
@@ -74,6 +74,7 @@ import org.smartregister.fhircore.quest.util.extensions.handleClickEvent
 import org.smartregister.p2p.utils.capitalize
 
 const val DIVIDER_TEST_TAG = "dividerTestTag"
+const val NUMBER_OF_ICONS_DISPLAYED = 3
```

Comment:
:+1: on extracting the number to a variable.  it makes it clear what the number represents